examples/examples/ws_notify_on_method_answered.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019-2021 Parity Technologies (UK) Ltd.
```

Comment:
nit: Maybe copyright years need to change

Reply:
let's take of it in another PR